File: Java/src/main/java/org/sisuns/bucles_while/intermedio/Ejercicio_05.java
package org.sisuns.bucles_while.intermedio;

public class Ejercicio_05 {
    public static void main(String[] args) {
        // Declaración de variables
        int suma = 0;
        int numero = 1;

        // Calcular la suma de cuadrados de pares y cubos de impares con un bucle while
        while (numero <= 10) {
            if (numero % 2 == 0) {
                // Sumar el cuadrado de números pares
                suma += Math.pow(numero, 2);
            } else {
                // Sumar el cubo de números impares
                suma += Math.pow(numero, 3);
            }
            numero++;
        }

        // Mostrar el resultado
        System.out.println("La suma de cuadrados de pares y cubos de impares entre 1 y 10 es: " + suma);
    }
}

File: Python/Bucles_while/Intermedio/Ejercicio_05.py
# Inicialización de variables
suma = 0
numero = 1

# Calcular la suma de cuadrados de pares y cubos de impares con un bucle Mientras
while numero <= 10:
    if numero % 2 == 0:
        # Sumar el cuadrado de números pares
        suma += numero**2
    else:
        # Sumar el cubo de números impares
        suma += numero**3
    numero += 1

# Mostrar el resultado
print("La suma de cuadrados de pares y cubos de impares entre 1 y 10 es:", suma)
